src/vis/scatter/utils.ts

```diff
@@ -26,6 +26,18 @@ export function isScatter(s: IVisConfig): s is IScatterConfig {
   return s.type === ESupportedPlotlyVis.SCATTER;
 }
 
+function calculateDomain(domain: [number | undefined, number | undefined], vals: number[]): [number, number] {
+  if (!domain || (domain[0] !== undefined && domain[1] !== undefined)) {
```

Comment:
@dvzacharycutler  This check can't be correct? I.e. if domain is null, you would still return `[domain[0], domain[1]]`?